src/tests/Dictionary/DictionaryModal.test.jsx

```diff
@@ -27,8 +27,17 @@ const props = {
 };
 
 describe('Test suite for dictionary modal', () => {
-  const wrapper = shallow(<DictionaryModal {...props} />);
-  const preventDefault = { preventDefault: jest.fn() };
+  let wrapper;
+  let preventDefault;
+
+  beforeEach(() => {
```

Comment:
I had refactored the structure of the tests to align with the coding convention

Reply:
Can you do that in a separate ticket and hence pull request?

Reply:
Sure, I'm doing that. Thank you.

Reply:
Share the ticket link here.

Reply:
Update: The PR is up and ready [here](https://github.com/openmrs/openmrs-ocl-client/pull/465)

Reply:
Did you take a look at this? https://github.com/openmrs/openmrs-ocl-client/pull/460/files

Reply:
Yes, I have rebased off [#465](https://github.com/openmrs/openmrs-ocl-client/pull/460) since I had to write a test for my changes with the proper structure. They'll not show up when #465 is merged

Reply:
Can you now explain this? https://github.com/openmrs/openmrs-ocl-client/pull/460/files

Reply:
Seems I have to rebase again from the master branch, doing so in a minute

Reply:
It's now fixed